LiskHQ/lisk-hub
# Set max amount - Closes #1234

### What issue have I solved?

-- #1234

### How have I implemented/fixed it?

Add a link (behaves like a button) that sets the input to max amount minus the transaction fee 

### How has this been tested?

1. Go to wallet send
2. Locate the amount input (bottom one)
3. Click the Set max. amount link
4. Input should have value of total LISK in wallet minus the transaction fee

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
